src/components/FindMovie/FindMovie.tsx

```diff
@@ -1,10 +1,62 @@
-import React from 'react';
+import React, { ChangeEvent, useState } from 'react';
+import cn from 'classnames';
+import { getMovie } from '../../api';
 import './FindMovie.scss';
+import { Movie } from '../../types/Movie';
+import { MovieCard } from '../MovieCard';
+import { createMovie } from '../../utils/createMovie';
+
+type FindMovieProps = {
+  onMovieAdd: (movie: Movie) => void;
+};
+
+export const FindMovie: React.FC<FindMovieProps> = ({ onMovieAdd }) => {
+  const [query, setQuery] = useState('');
+  const [searchedMovie, setSearchedMovie] = useState<Movie | null>(null);
+  const [hasError, setHasError] = useState(false);
+  const [isLoading, setIsLoading] = useState(false);
+
+  const handleFindMovie = async () => {
+    setIsLoading(true);
+    const movieData = await getMovie(query);
+
+    if ('Error' in movieData) {
```

Comment:
Use catch for errors in [`try {...} catch (e) {...}`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/try...catch)